includes/features/SU-SWS/stanford_private_page/stanford_private_page.feature

```diff
@@ -16,6 +16,35 @@ Feature: Stanford Private Page
     Then I click "Edit" in the "Tabs" region
     Then I press "Save"
     And I should see "Bar" in the "Content Head" region
+    And I should be on "private/bar"
+    And I click "Edit" in the "Content Head" region
+    And I click on the element with css selector "#edit-delete"
+    And I click on the element with css selector "#edit-submit"
+    Then I should see the text "Private Page Bar has been deleted" in the "Console" region
+
+
+    @api @dev @destructive
+    Scenario: Adminstrator can upload a file and anonymous user cannot access file
+      Given I am logged in as a user with the "administrator" role
+      When I go to "node/add/stanford-private-page"
+      Then I should see "Create Private Page" in the "Branding" region
+      Then I fill in "edit-title" with "Bar"
+      Then I attach the file "files/sample.pdf" to "edit-field-s-private-file-und-0-upload"
+      Then I press "Upload"
+      Then I press "Save"
+      Given I am an anonymous user
+      And I am on the homepage
+      Then I go to "system/files/sample.pdf"
+      Then I should see "Access denied"
```

Comment:
@cjwest Let me know if I might have missed a setup step.  This is the only test that failed.  On sws-build-jse, I am seeing the following at system/files/sample.pdf:

**Logged in**
![screen shot 2018-01-31 at 12 33 38 pm](https://user-images.githubusercontent.com/4072139/35646578-7a31dcbc-0684-11e8-904a-dada1feee014.png)

**Logged out**
![screen shot 2018-01-31 at 12 34 15 pm](https://user-images.githubusercontent.com/4072139/35646595-84d1c952-0684-11e8-89d0-f4a7b21c6810.png)

And the following at sites/default/files/sample.pdf:

**Logged in**
![screen shot 2018-01-31 at 12 33 50 pm](https://user-images.githubusercontent.com/4072139/35646635-9fdf47e2-0684-11e8-8811-a9bd31777e89.png)

**Logged out (same)**
![screen shot 2018-01-31 at 12 34 27 pm](https://user-images.githubusercontent.com/4072139/35646647-a7de45f6-0684-11e8-8deb-21429576d55b.png)

Reply:
Database update was required.

Reply:
This step is failing on sites environment because anonymous users are sent to Webauth instead of hitting the Access denied page.

Reply:
Hi @kbrownell,

I put in a fix, and added a step for testing that anonymous users _can't_ see images as well.
*Warning*: One problem I'm seeing is that if you've looked at the image or file in your browser the test will fail. You probably need to clear the cache on the browser.

Reply:
This looks great!  Thanks for reworking that piece.